react/features/base/react/components/web/NavigateSectionListItem.js

```diff
@@ -68,6 +85,7 @@ export default class NavigateSectionListItem extends Component<Props> {
                     className = 'navigate-section-tile-body'>
                     { duration }
                 </Text>
```

Comment:
Is NavigateSectionListItem specifically for calendar entries or do you know if it was intended to be something more general? It's markup makes it look like it was specific for calendar entries.